app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -20,81 +24,109 @@ import DeviceContext from '../context/device';
 import UploadContext from '../context/upload';
 import useCounter from '../hooks/use-counter';
 import useCookie from '../hooks/use-cookie';
+import type {
+  AcuantSuccessResponse,
+  AcuantDocumentType,
+  AcuantCaptureFailureError,
+} from './acuant-camera';
 
-/** @typedef {import('react').ReactNode} ReactNode */
-/** @typedef {import('./acuant-camera').AcuantSuccessResponse} AcuantSuccessResponse */
-/** @typedef {import('./acuant-camera').AcuantDocumentType} AcuantDocumentType */
-/** @typedef {import('@18f/identity-components').FullScreenRefHandle} FullScreenRefHandle */
-/** @typedef {import('../context/acuant').AcuantGlobal} AcuantGlobal */
+type AcuantDocumentTypeLabel = 'id' | 'passport' | 'none';
+type AcuantImageAssessment = 'success' | 'glare' | 'blurry';
+type ImageSource = 'acuant' | 'upload';
 
-/**
- * @typedef {"id"|"passport"|"none"} AcuantDocumentTypeLabel
- */
-
-/**
- * @typedef {"success"|"glare"|"blurry"} AcuantImageAssessment
- */
-
-/**
- * @typedef {"acuant"|"upload"} ImageSource
- */
-
-/**
- * @typedef ImageAnalyticsPayload
- *
- * @prop {number?} width Image width, or null if unknown.
- * @prop {number?} height Image height, or null if unknown.
- * @prop {string?} mimeType Mime type, or null if unknown.
- * @prop {ImageSource} source Method by which image was added.
- * @prop {number=} attempt Total number of attempts at this point.
- * @prop {number} size Size of image, in bytes.
- */
+interface ImageAnalyticsPayload {
+  /**
+   * Image width, or null if unknown
+   */
+  width?: number | null;
```

Comment:
Before, these were `number | null`, but not optional.

```suggestion
  width: number | null;
```

Reply:
Thanks, I flipped the original non-optionals back to being non-optional in that interface https://github.com/18F/identity-idp/pull/6769/commits/8d31f0a628d7cabc3c73f6c1f804d1ff4d5d7ceb